src/Pages/Home/index.jsx

```diff
@@ -52,7 +52,7 @@ export default function Home() {
 
   useEffect(() => {
     axios
-      .get(`/home`)
+      .get(`http://localhost:3031/news`)
```

Comment:
Don't forget to hide your localhost into a .env file

Reply:
Next PR